src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/upstart/start-template

```diff
@@ -27,7 +27,11 @@ end script
 # set the working directory of the job processes
 chdir ${{chdir}}
 
+setuid ${{daemon_user}}
```

Comment:
I think we had that at one point in time and it didn't work for some reason. On which systems did you test this?